sonar-csharp-plugin/src/main/java/org/sonar/plugins/csharp/CSharpSonarWayProfile.java

```diff
@@ -44,14 +43,8 @@ public void define(Context context) {
       Class<?> csRulesClass = Class.forName("com.sonar.plugins.security.api.CsRules");
       Method getRuleKeysMethod = csRulesClass.getMethod("getRuleKeys");
       return (Set<String>) getRuleKeysMethod.invoke(null);
-    } catch (ClassNotFoundException e) {
-      LOG.debug("com.sonar.plugins.security.api.CsRules is not found, no security rules added to Sonar way cs profile: " + e.getMessage());
-    } catch (NoSuchMethodException e) {
-      LOG.debug("com.sonar.plugins.security.api.CsRules#getRuleKeys is not found, no security rules added to Sonar way cs profile: " + e.getMessage());
-    } catch (IllegalAccessException e) {
-      LOG.debug("[IllegalAccessException] no security rules added to Sonar way cs profile: " + e.getMessage());
-    } catch (InvocationTargetException e) {
-      LOG.debug("[InvocationTargetException] no security rules added to Sonar way cs profile: " + e.getMessage());
+    } catch (Exception e) {
+      LOG.debug("[Exception] no security rules added to Sonar way cs profile: " + e.getMessage());
```

Comment:
Could you log the class of the exception, just to preserve a similar level of detail as the previous messages? e.g. _e.getClass().getCanonicalName()_

Reply:
FYI looks like you can catch multiple exceptions in a single _catch_, as a alternative to catching _Exception_
i.e. catch (ClassNotFoundException|NoSuchMethodException| etc... )